src/components/facet-browser/helper.ts

```diff
@@ -132,8 +134,29 @@ export const getFacetFieldTranslation = (name: FacetField) => {
   }
 };
 
+export const createFacetsMap = (
```

Comment:
I am bad at remembering this myself. But I think it is very valuable to write a small comment about what the purpose of a function is. Unless it is self-explanatory, eg.: getUrlQueryParams()
So in this case: "what is a facet map and why is it needed?"